docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -122,6 +122,18 @@ It is recommended you do a [backup]({{< relref "doc/usage/backup-and-restore.en-
 If you have carried out the installation steps as described above, the binary should
 have the generic name `gitea`. Do not change this, i.e. to include the version number.
 
+### 1. Use systemd (recommended)
+
+As explained before, we recommended to to use service system as systemd. In this case ```systemctl restart gitea``` should be enough.
+
+### 2. Use command-line/terminal
```

Comment:
Strictly speaking, `### 1. Use systemd (recommended)` above is also using command line in terminal.

I suggest changing this header to `### 2. Restarting gitea without systemd`